packages/emotion/src/EmotionThemeProvider/README.md

```diff
@@ -2,6 +2,10 @@
 describes: EmotionThemeProvider
 ---
 
+#### DEPRECATED. Please use [InstUISettingsProvider](#InstUISettingsProvider) instead. IT has the same functionality and
+
+adds text direction configuration.
+
```

Comment:
I think this sentence should be in the same heading(meaning I wouldn't break the text at the `and` word) because it looks kinda weird rendered:

<img width="1041" alt="Screenshot 2021-11-08 at 10 43 20" src="https://user-images.githubusercontent.com/20640237/140719365-430ad473-e54e-476f-8309-58fc791d2954.png">

Reply:
fixed